Build error on 2.8.0

I'm getting a build error on 2.8.0 (in https://github.com/alphagov/content-store). It looks related to the rb-sys bump. Selected output below

```
Gem::Ext::BuildError: ERROR: Failed to build gem native extension.
```

```
error: failed to run custom build command for `rb-sys v0.9.126`
Caused by:
  process didn't exit successfully: `/root/.rbenv/versions/3.4.4/lib/ruby/gems/3.4.0/gems/commonmarker-2.8.0/ext/commonmarker/target/release/build/rb-sys-bed89f6b70d35e6f/build-script-main` (exit status: 101)
```

```
  thread 'main' (1615) panicked at /root/.rbenv/versions/3.4.4/lib/ruby/gems/3.4.0/gems/commonmarker-2.8.0/ext/commonmarker/.rb-sys/stable/cargo/registry/src/index.crates.io-1949cf8c6b5b557f/bindgen-0.72.1/lib.rs:616:27:
Unable to find libclang: "couldn't find any valid shared libraries matching: ['libclang.so', 'libclang-*.so', 'libclang.so.*', 'libclang-*.so.*'], set the `LIBCLANG_PATH` environment variable to a path where one of these files can be
found (invalid: [])"
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
make: *** [Makefile:573: target/release/libcommonmarker.so] Error 101

make failed, exit code 2
```

```
An error occurred while installing commonmarker (2.8.0), and Bundler cannot continue.
```

This was via `bundle update` and it failed so there's no committed Gemfile.lock, but the full log from bundler is below

<details>
<summary>Full log</summary>

```
Gem::Ext::BuildError: ERROR: Failed to build gem native extension.

    current directory: /root/.rbenv/versions/4.0.2/lib/ruby/gems/4.0.0/gems/commonmarker-2.8.0/ext/commonmarker
/root/.rbenv/versions/4.0.2/bin/ruby extconf.rb
checking for gcc... yes
checking for g++... yes
checking for gcc-ar... yes
checking for cargo... no
checking for install_name_tool... no
checking for $(CARGO_BUILD_TARGET)-install_name_tool... no

current directory: /root/.rbenv/versions/4.0.2/lib/ruby/gems/4.0.0/gems/commonmarker-2.8.0/ext/commonmarker
make -j9 DESTDIR\= sitearchdir\=./.gem.20260413-1-2jwib7 sitelibdir\=./.gem.20260413-1-2jwib7 clean

current directory: /root/.rbenv/versions/4.0.2/lib/ruby/gems/4.0.0/gems/commonmarker-2.8.0/ext/commonmarker
make -j9 DESTDIR\= sitearchdir\=./.gem.20260413-1-2jwib7 sitelibdir\=./.gem.20260413-1-2jwib7
info: downloading installer
warn: It looks like you have an existing rustup settings file at:
warn: /root/.rbenv/versions/4.0.2/lib/ruby/gems/4.0.0/gems/commonmarker-2.8.0/ext/commonmarker/.rb-sys/stable/rustup/settings.toml
warn: Rustup will install the default toolchain as specified in the settings file,
warn: instead of the one inferred from the default host triple.
info: profile set to minimal
info: default host triple is aarch64-unknown-linux-gnu
info: skipping toolchain installation


Rust is installed now. Great!

To get started you need Cargo's bin directory
(/root/.rbenv/versions/4.0.2/lib/ruby/gems/4.0.0/gems/commonmarker-2.8.0/ext/commonmarker/.rb-sys/stable/cargo/bin)
in your PATH
environment variable. This has not been done automatically.

To configure your current shell, you need to source
the corresponding env file under
/root/.rbenv/versions/4.0.2/lib/ruby/gems/4.0.0/gems/commonmarker-2.8.0/ext/commonmarker/.rb-sys/stable/cargo.

This is usually done by running one of the following (note the leading DOT):
.
"/root/.rbenv/versions/4.0.2/lib/ruby/gems/4.0.0/gems/commonmarker-2.8.0/ext/commonmarker/.rb-sys/stable/cargo/env"
# For sh/bash/zsh/ash/dash/pdksh
source
"/root/.rbenv/versions/4.0.2/lib/ruby/gems/4.0.0/gems/commonmarker-2.8.0/ext/commonmarker/.rb-sys/stable/cargo/env.fish"
# For fish
source
"/root/.rbenv/versions/4.0.2/lib/ruby/gems/4.0.0/gems/commonmarker-2.8.0/ext/commonmarker/.rb-sys/stable/cargo/env.nu"
# For nushell
source
"/root/.rbenv/versions/4.0.2/lib/ruby/gems/4.0.0/gems/commonmarker-2.8.0/ext/commonmarker/.rb-sys/stable/cargo/env.tcsh"
# For tcsh
.
"/root/.rbenv/versions/4.0.2/lib/ruby/gems/4.0.0/gems/commonmarker-2.8.0/ext/commonmarker/.rb-sys/stable/cargo/env.ps1"
# For pwsh
source
"/root/.rbenv/versions/4.0.2/lib/ruby/gems/4.0.0/gems/commonmarker-2.8.0/ext/commonmarker/.rb-sys/stable/cargo/env.xsh"
# For xonsh
info: syncing channel updates for stable-aarch64-unknown-linux-gnu
info: latest update on 2026-03-26 for version 1.94.1 (e408947bf 2026-03-25)
info: downloading 3 components

  stable-aarch64-unknown-linux-gnu installed - rustc 1.94.1 (e408947bf 2026-03-25)

info: default toolchain set to stable-aarch64-unknown-linux-gnu
info: checking for self-update (current version: 1.29.0)
info: using existing install for stable-aarch64-unknown-linux-gnu
info: default toolchain set to stable-aarch64-unknown-linux-gnu

  stable-aarch64-unknown-linux-gnu unchanged - rustc 1.94.1 (e408947bf 2026-03-25)

info: note that the toolchain 'stable-aarch64-unknown-linux-gnu' is currently in use (overridden by environment variable RUSTUP_TOOLCHAIN)
generating target/release/libcommonmarker.so (release)
/root/.rbenv/versions/4.0.2/lib/ruby/gems/4.0.0/gems/commonmarker-2.8.0/ext/commonmarker/.rb-sys/stable/cargo/bin/cargo rustc  --manifest-path ./Cargo.toml --target-dir target --lib --profile release -- -C linker=gcc -L
native=/root/.rbenv/versions/4.0.2/lib -C link-arg=-lm -l pthread
    Updating crates.io index
 Downloading crates ...
  Downloaded time-core v0.1.8
  Downloaded tinyvec_macros v0.1.1
  Downloaded typed-arena v2.0.2
  Downloaded utf8parse v0.2.2
  Downloaded terminal_size v0.4.3
  Downloaded thiserror v2.0.18
  Downloaded xdg v3.0.0
  Downloaded time-macros v0.2.27
  Downloaded walkdir v2.5.0
  Downloaded zmij v1.0.15
  Downloaded thiserror-impl v2.0.18
  Downloaded tinyvec v1.10.0
  Downloaded yaml-rust v0.4.5
  Downloaded unicode-ident v1.0.22
  Downloaded unicode-normalization v0.1.25
  Downloaded time v0.3.47
  Downloaded syn v2.0.114
  Downloaded syntect v5.3.0
  Downloaded darling_macro v0.23.0
  Downloaded fmt2io v1.0.0
  Downloaded ident_case v1.0.1
  Downloaded magnus-macros v0.8.0
  Downloaded colorchoice v1.0.4
  Downloaded equivalent v1.0.2
  Downloaded is_terminal_polyfill v1.70.2
  Downloaded rctree v0.6.0
  Downloaded num-conv v0.2.0
  Downloaded rb-sys-env v0.2.3
  Downloaded cfg-if v1.0.4
  Downloaded rb-allocator v0.9.6
  Downloaded adler2 v2.0.1
  Downloaded fnv v1.0.7
  Downloaded lazy_static v1.5.0
  Downloaded siphasher v1.0.1
  Downloaded shell-words v1.1.1
  Downloaded fastrand v2.3.0
  Downloaded phf_generator v0.13.1
  Downloaded same-file v1.0.6
  Downloaded anstyle v1.0.13
  Downloaded anstyle-query v1.1.5
  Downloaded linked-hash-map v0.5.6
  Downloaded strsim v0.11.1
  Downloaded phf_shared v0.13.1
  Downloaded heck v0.5.0
  Downloaded rustc-hash v2.1.1
  Downloaded errno v0.3.14
  Downloaded seq-macro v0.3.6
  Downloaded caseless v0.2.2
  Downloaded either v1.15.0
  Downloaded itoa v1.0.17
  Downloaded phf_codegen v0.13.1
  Downloaded bit-set v0.8.0
  Downloaded powerfmt v0.2.0
  Downloaded pkg-config v0.3.32
  Downloaded clap_lex v0.7.7
  Downloaded shlex v1.3.0
  Downloaded bit-vec v0.8.0
  Downloaded rb-sys-build v0.9.126
  Downloaded anstyle-parse v0.2.7
  Downloaded cexpr v0.6.0
  Downloaded deranged v0.5.5
  Downloaded find-msvc-tools v0.1.8
  Downloaded simd-adler32 v0.3.8
  Downloaded phf v0.13.1
  Downloaded jetscii v0.5.3
  Downloaded rustversion v1.0.22
  Downloaded glob v0.3.3
  Downloaded anstream v0.6.21
  Downloaded entities v1.0.1
  Downloaded bincode v1.3.3
  Downloaded clap_derive v4.5.49
  Downloaded libloading v0.8.9
  Downloaded darling v0.23.0
  Downloaded crc32fast v1.5.0
  Downloaded onig v6.5.1
  Downloaded smallvec v1.15.1
  Downloaded once_cell v1.21.3
  Downloaded bitflags v2.10.0
  Downloaded clang-sys v1.8.1
  Downloaded rb-sys v0.9.126
  Downloaded quote v1.0.43
  Downloaded proc-macro2 v1.0.105
  Downloaded clap v4.5.54
  Downloaded serde_derive v1.0.228
  Downloaded finl_unicode v1.4.0
  Downloaded plist v1.8.0
  Downloaded miniz_oxide v0.8.9
  Downloaded serde_core v1.0.228
  Downloaded bon v3.8.2
  Downloaded flate2 v1.1.8
  Downloaded darling_core v0.23.0
  Downloaded prettyplease v0.2.37
  Downloaded serde v1.0.228
  Downloaded bon-macros v3.8.2
  Downloaded fancy-regex v0.16.2
  Downloaded cc v1.2.53
  Downloaded base64 v0.22.1
  Downloaded minimal-lexical v0.2.1
  Downloaded memchr v2.7.6
  Downloaded indexmap v2.13.0
  Downloaded nom v7.1.3
  Downloaded itertools v0.12.1
  Downloaded hashbrown v0.16.1
  Downloaded emojis v0.8.0
  Downloaded clap_builder v4.5.54
  Downloaded serde_json v1.0.149
  Downloaded aho-corasick v1.1.4
  Downloaded regex v1.12.2
  Downloaded magnus v0.8.2
  Downloaded quick-xml v0.38.4
  Downloaded bindgen v0.72.1
  Downloaded comrak v0.52.0
  Downloaded regex-syntax v0.8.8
  Downloaded rustix v1.1.3
  Downloaded regex-automata v0.4.13
  Downloaded onig_sys v69.9.1
  Downloaded libc v0.2.180
  Downloaded linux-raw-sys v0.11.0
   Compiling proc-macro2 v1.0.105
   Compiling quote v1.0.43
   Compiling unicode-ident v1.0.22
   Compiling shlex v1.3.0
   Compiling memchr v2.7.6
   Compiling glob v0.3.3
   Compiling libc v0.2.180
   Compiling minimal-lexical v0.2.1
   Compiling cfg-if v1.0.4
   Compiling regex-syntax v0.8.8
   Compiling libloading v0.8.9
   Compiling bindgen v0.72.1
   Compiling either v1.15.0
   Compiling rustc-hash v2.1.1
   Compiling bitflags v2.10.0
   Compiling clang-sys v1.8.1
   Compiling aho-corasick v1.1.4
   Compiling nom v7.1.3
   Compiling itertools v0.12.1
   Compiling serde_core v1.0.228
   Compiling shell-words v1.1.1
   Compiling syn v2.0.114
   Compiling lazy_static v1.5.0
   Compiling find-msvc-tools v0.1.8
   Compiling cc v1.2.53
   Compiling strsim v0.11.1
   Compiling ident_case v1.0.1
   Compiling cexpr v0.6.0
   Compiling pkg-config v0.3.32
   Compiling regex-automata v0.4.13
   Compiling serde v1.0.228
   Compiling rustix v1.1.3
   Compiling siphasher v1.0.1
   Compiling phf_shared v0.13.1
   Compiling onig_sys v69.9.1
   Compiling utf8parse v0.2.2
   Compiling linux-raw-sys v0.11.0
   Compiling crc32fast v1.5.0
   Compiling fastrand v2.3.0
   Compiling rb-sys-env v0.2.3
   Compiling itoa v1.0.17
   Compiling zmij v1.0.15
   Compiling prettyplease v0.2.37
   Compiling rustversion v1.0.22
   Compiling powerfmt v0.2.0
   Compiling phf_generator v0.13.1
   Compiling deranged v0.5.5
   Compiling anstyle-parse v0.2.7
   Compiling anstyle v1.0.13
   Compiling regex v1.12.2
   Compiling darling_core v0.23.0
   Compiling adler2 v2.0.1
   Compiling equivalent v1.0.2
   Compiling hashbrown v0.16.1
   Compiling simd-adler32 v0.3.8
   Compiling colorchoice v1.0.4
   Compiling anstyle-query v1.1.5
   Compiling is_terminal_polyfill v1.70.2
   Compiling tinyvec_macros v0.1.1
   Compiling num-conv v0.2.0
   Compiling time-core v0.1.8
   Compiling serde_json v1.0.149
   Compiling thiserror v2.0.18
   Compiling time v0.3.47
   Compiling indexmap v2.13.0
   Compiling tinyvec v1.10.0
   Compiling anstream v0.6.21
   Compiling miniz_oxide v0.8.9
   Compiling thiserror-impl v2.0.18
   Compiling terminal_size v0.4.3
   Compiling phf_codegen v0.13.1
   Compiling quick-xml v0.38.4
   Compiling same-file v1.0.6
   Compiling once_cell v1.21.3
   Compiling darling_macro v0.23.0
   Compiling jetscii v0.5.3
   Compiling entities v1.0.1
   Compiling clap_lex v0.7.7
   Compiling linked-hash-map v0.5.6
   Compiling base64 v0.22.1
   Compiling heck v0.5.0
   Compiling darling v0.23.0
   Compiling bon-macros v3.8.2
   Compiling clap_derive v4.5.49
   Compiling yaml-rust v0.4.5
   Compiling clap_builder v4.5.54
   Compiling rb-sys-build v0.9.126
   Compiling plist v1.8.0
   Compiling comrak v0.52.0
   Compiling magnus v0.8.2
   Compiling walkdir v2.5.0
   Compiling flate2 v1.1.8
   Compiling serde_derive v1.0.228
   Compiling unicode-normalization v0.1.25
   Compiling phf v0.13.1
   Compiling rb-sys v0.9.126
   Compiling bincode v1.3.3
   Compiling fnv v1.0.7
   Compiling emojis v0.8.0
   Compiling caseless v0.2.2
   Compiling clap v4.5.54
   Compiling magnus-macros v0.8.0
   Compiling commonmarker v1.0.0 (/root/.rbenv/versions/4.0.2/lib/ruby/gems/4.0.0/gems/commonmarker-2.8.0/ext/commonmarker)
   Compiling bon v3.8.2
   Compiling typed-arena v2.0.2
   Compiling seq-macro v0.3.6
   Compiling smallvec v1.15.1
   Compiling finl_unicode v1.4.0
   Compiling xdg v3.0.0
   Compiling fmt2io v1.0.0
   Compiling rctree v0.6.0
error: failed to run custom build command for `rb-sys v0.9.126`

Caused by:
  process didn't exit successfully: `/root/.rbenv/versions/4.0.2/lib/ruby/gems/4.0.0/gems/commonmarker-2.8.0/ext/commonmarker/target/release/build/rb-sys-bed89f6b70d35e6f/build-script-main` (exit status: 101)
  --- stdout
  cargo:rerun-if-env-changed=RUBY
  cargo:rerun-if-env-changed=RBCONFIG_CROSS_COMPILING
  cargo:rerun-if-env-changed=RBCONFIG_RUBY_PROGRAM_VERSION
  cargo:rerun-if-env-changed=RBCONFIG_ruby_install_name
  cargo:rerun-if-env-changed=RBCONFIG_platform
  cargo:rerun-if-env-changed=RUBY_ROOT
  cargo:rerun-if-env-changed=RUBY_VERSION
  cargo:rerun-if-env-changed=RUBY
  cargo:rerun-if-changed=build/features.rs
  cargo:rerun-if-changed=build/stable_api_config.rs
  cargo:rerun-if-changed=build/main.rs
  cargo:rerun-if-changed=build/version.rs
  cargo:rerun-if-env-changed=RUBY_STATIC
  cargo:rerun-if-env-changed=RBCONFIG_ENABLE_SHARED
  cargo:rerun-if-env-changed=RBCONFIG_rubyhdrdir
  cargo:rerun-if-env-changed=RBCONFIG_rubyarchhdrdir
  cargo:rerun-if-env-changed=RBCONFIG_CPPFLAGS
  cargo:rerun-if-env-changed=RBCONFIG_rubyhdrdir
  cargo:rerun-if-env-changed=RBCONFIG_MAJOR
  cargo:rerun-if-env-changed=RBCONFIG_MINOR
  cargo:rerun-if-env-changed=TARGET
  cargo:rerun-if-env-changed=TARGET
  cargo:rerun-if-env-changed=BINDGEN_EXTRA_CLANG_ARGS_aarch64-unknown-linux-gnu
  cargo:rerun-if-env-changed=BINDGEN_EXTRA_CLANG_ARGS_aarch64-unknown-linux-gnu
  cargo:rerun-if-env-changed=BINDGEN_EXTRA_CLANG_ARGS_aarch64_unknown_linux_gnu
  cargo:rerun-if-env-changed=BINDGEN_EXTRA_CLANG_ARGS_aarch64_unknown_linux_gnu
  cargo:rerun-if-env-changed=BINDGEN_EXTRA_CLANG_ARGS
  cargo:rerun-if-env-changed=BINDGEN_EXTRA_CLANG_ARGS

  --- stderr
INFO: using bindgen with clang args: ["-I/root/.rbenv/versions/4.0.2/include/ruby-4.0.0", "-I/root/.rbenv/versions/4.0.2/include/ruby-4.0.0/aarch64-linux", "-fms-extensions", "-mbranch-protection=pac-ret", "-fstack-protector-strong",
"-U_FORTIFY_SOURCE", "-D_FORTIFY_SOURCE=2", "-O3", "-fno-fast-math", "-ggdb3", "-Wall", "-Wextra", "-Wdeprecated-declarations", "-Wdiv-by-zero", "-Wduplicated-cond", "-Wimplicit-function-declaration", "-Wimplicit-int", "-Wpointer-arith",
"-Wwrite-strings", "-Wold-style-definition", "-Wimplicit-fallthrough=0", "-Wmissing-noreturn", "-Wno-cast-function-type", "-Wno-constant-logical-operand", "-Wno-long-long", "-Wno-missing-field-initializers", "-Wno-overlength-strings",
"-Wno-packed-bitfield-compat", "-Wno-parentheses-equality", "-Wno-self-assign", "-Wno-tautological-compare", "-Wno-unused-parameter", "-Wno-unused-value", "-Wsuggest-attribute=format", "-Wsuggest-attribute=noreturn", "-Wunused-variable",
"-Wmisleading-indentation", "-Wundef"]
  #include "ruby.h"

  #ifdef HAVE_RUBY_DEBUG_H
  #include "ruby/debug.h"
  #endif
  #ifdef HAVE_RUBY_DEFINES_H
  #include "ruby/defines.h"
  #endif
  #ifdef HAVE_RUBY_ENCODING_H
  #include "ruby/encoding.h"
  #endif
  #ifdef HAVE_RUBY_FIBER_SCHEDULER_H
  #include "ruby/fiber/scheduler.h"
  #endif
  #ifdef HAVE_RUBY_INTERN_H
  #include "ruby/intern.h"
  #endif
  #ifdef HAVE_RUBY_IO_H
  #include "ruby/io.h"
  #endif
  #ifdef HAVE_RUBY_MEMORY_VIEW_H
  #include "ruby/memory_view.h"
  #endif
  #ifdef HAVE_RUBY_MISSING_H
  #include "ruby/missing.h"
  #endif
  #ifdef HAVE_RUBY_ONIGMO_H
  #include "ruby/onigmo.h"
  #endif
  #ifdef HAVE_RUBY_ONIGURUMA_H
  #include "ruby/oniguruma.h"
  #endif
  #ifdef HAVE_RUBY_RACTOR_H
  #include "ruby/ractor.h"
  #endif
  #ifdef HAVE_RUBY_RANDOM_H
  #include "ruby/random.h"
  #endif
  #ifdef HAVE_RUBY_RE_H
  #include "ruby/re.h"
  #endif
  #ifdef HAVE_RUBY_REGEX_H
  #include "ruby/regex.h"
  #endif
  #ifdef HAVE_RUBY_RUBY_H
  #include "ruby/ruby.h"
  #endif
  #ifdef HAVE_RUBY_ST_H
  #include "ruby/st.h"
  #endif
  #ifdef HAVE_RUBY_THREAD_H
  #include "ruby/thread.h"
  #endif
  #ifdef HAVE_RUBY_THREAD_NATIVE_H
  #include "ruby/thread_native.h"
  #endif
  #ifdef HAVE_RUBY_UTIL_H
  #include "ruby/util.h"
  #endif
  #ifdef HAVE_RUBY_VERSION_H
  #include "ruby/version.h"
  #endif
  #ifdef HAVE_RUBY_VM_H
  #include "ruby/vm.h"
  #endif
  #ifdef HAVE_RUBY_WIN32_H
  #include "ruby/win32.h"
  #endif
  #ifdef HAVE_RUBY_IO_BUFFER_H
  #include "ruby/io/buffer.h"
  #endif
  #ifdef HAVE_RUBY_ATOMIC_H
  #include "ruby/atomic.h"
  #endif
  struct rb_sys__Opaque__RString { struct RString dummy; };
  struct rb_sys__Opaque__RArray { struct RArray dummy; };
  struct rb_sys__Opaque__RData { struct RData dummy; };
  struct rb_sys__Opaque__RTypedData { struct RTypedData dummy; };
  struct rb_sys__Opaque__rb_matchext_struct { struct rb_matchext_struct dummy; };
  struct rb_sys__Opaque__rb_internal_thread_event_data { struct rb_internal_thread_event_data dummy; };
  struct rb_sys__Opaque__rb_io_internal_buffer { struct rb_io_internal_buffer dummy; };

  thread 'main' (1698) panicked at /root/.rbenv/versions/4.0.2/lib/ruby/gems/4.0.0/gems/commonmarker-2.8.0/ext/commonmarker/.rb-sys/stable/cargo/registry/src/index.crates.io-1949cf8c6b5b557f/bindgen-0.72.1/lib.rs:616:27:
Unable to find libclang: "couldn't find any valid shared libraries matching: ['libclang.so', 'libclang-*.so', 'libclang.so.*', 'libclang-*.so.*'], set the `LIBCLANG_PATH` environment variable to a path where one of these files can be
found (invalid: [])"
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
make: *** [Makefile:581: target/release/libcommonmarker.so] Error 101

make failed, exit code 2

Gem files will remain installed in /root/.rbenv/versions/4.0.2/lib/ruby/gems/4.0.0/gems/commonmarker-2.8.0 for inspection.
Results logged to /root/.rbenv/versions/4.0.2/lib/ruby/gems/4.0.0/extensions/aarch64-linux/4.0.0/commonmarker-2.8.0/gem_make.out

  /root/.rbenv/versions/4.0.2/lib/ruby/4.0.0/rubygems/ext/builder.rb:139:in 'Gem::Ext::Builder.run'
  /root/.rbenv/versions/4.0.2/lib/ruby/4.0.0/rubygems/ext/builder.rb:65:in 'block in Gem::Ext::Builder.make'
  /root/.rbenv/versions/4.0.2/lib/ruby/4.0.0/rubygems/ext/builder.rb:57:in 'Array#each'
  /root/.rbenv/versions/4.0.2/lib/ruby/4.0.0/rubygems/ext/builder.rb:57:in 'Gem::Ext::Builder.make'
  /root/.rbenv/versions/4.0.2/lib/ruby/4.0.0/rubygems/ext/ext_conf_builder.rb:44:in 'Gem::Ext::ExtConfBuilder.build'
  /root/.rbenv/versions/4.0.2/lib/ruby/4.0.0/rubygems/ext/builder.rb:218:in 'Gem::Ext::Builder#build_extension'
  /root/.rbenv/versions/4.0.2/lib/ruby/4.0.0/rubygems/ext/builder.rb:250:in 'block in Gem::Ext::Builder#build_extensions'
  /root/.rbenv/versions/4.0.2/lib/ruby/4.0.0/rubygems/ext/builder.rb:249:in 'Array#each'
  /root/.rbenv/versions/4.0.2/lib/ruby/4.0.0/rubygems/ext/builder.rb:249:in 'Gem::Ext::Builder#build_extensions'
  /root/.rbenv/versions/4.0.2/lib/ruby/4.0.0/rubygems/installer.rb:809:in 'Gem::Installer#build_extensions'
  /root/.rbenv/versions/4.0.2/lib/ruby/gems/4.0.0/gems/bundler-2.5.22/lib/bundler/rubygems_gem_installer.rb:109:in 'Bundler::RubyGemsGemInstaller#build_extensions'
  /root/.rbenv/versions/4.0.2/lib/ruby/gems/4.0.0/gems/bundler-2.5.22/lib/bundler/rubygems_gem_installer.rb:28:in 'Bundler::RubyGemsGemInstaller#install'
  /root/.rbenv/versions/4.0.2/lib/ruby/gems/4.0.0/gems/bundler-2.5.22/lib/bundler/source/rubygems.rb:205:in 'Bundler::Source::Rubygems#install'
  /root/.rbenv/versions/4.0.2/lib/ruby/gems/4.0.0/gems/bundler-2.5.22/lib/bundler/installer/gem_installer.rb:55:in 'Bundler::GemInstaller#install'
  /root/.rbenv/versions/4.0.2/lib/ruby/gems/4.0.0/gems/bundler-2.5.22/lib/bundler/installer/gem_installer.rb:17:in 'Bundler::GemInstaller#install_from_spec'
  /root/.rbenv/versions/4.0.2/lib/ruby/gems/4.0.0/gems/bundler-2.5.22/lib/bundler/installer/parallel_installer.rb:133:in 'Bundler::ParallelInstaller#do_install'
  /root/.rbenv/versions/4.0.2/lib/ruby/gems/4.0.0/gems/bundler-2.5.22/lib/bundler/installer/parallel_installer.rb:124:in 'block in Bundler::ParallelInstaller#worker_pool'
  /root/.rbenv/versions/4.0.2/lib/ruby/gems/4.0.0/gems/bundler-2.5.22/lib/bundler/worker.rb:62:in 'Bundler::Worker#apply_func'
  /root/.rbenv/versions/4.0.2/lib/ruby/gems/4.0.0/gems/bundler-2.5.22/lib/bundler/worker.rb:57:in 'block in Bundler::Worker#process_queue'
  /root/.rbenv/versions/4.0.2/lib/ruby/gems/4.0.0/gems/bundler-2.5.22/lib/bundler/worker.rb:54:in 'Kernel#loop'
  /root/.rbenv/versions/4.0.2/lib/ruby/gems/4.0.0/gems/bundler-2.5.22/lib/bundler/worker.rb:54:in 'Bundler::Worker#process_queue'
  /root/.rbenv/versions/4.0.2/lib/ruby/gems/4.0.0/gems/bundler-2.5.22/lib/bundler/worker.rb:90:in 'block (2 levels) in Bundler::Worker#create_threads'

An error occurred while installing commonmarker (2.8.0), and Bundler cannot continue.

In Gemfile:
  openapi3_parser was resolved to 0.10.1, which depends on
    commonmarker
```
</details>

--- a/lib/commonmarker/version.rb
+++ b/lib/commonmarker/version.rb
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Commonmarker
-  VERSION = "2.8.0"
+  VERSION = "2.8.1"
 end

